swagger-parser/2.0.20.wso2v1/pom.xml

```diff
@@ -187,7 +187,8 @@
         <commons.io.version.range>[2.6,3.0)</commons.io.version.range>
         <commons.lang.version.range>[3.2.1,4)</commons.lang.version.range>
         <snakeyaml.version.range>[1.20,1.30)</snakeyaml.version.range>
-        <jackson-version>2.10.3</jackson-version>
+        <jackson-version>2.13.4</jackson-version>
+        <jackson-databind-version>2.13.4</jackson-databind-version>
```

Comment:
We cannot edit the same orbit bundle in the public repo. Let's create a new one `2.0.20.wso2v2` and deploy

Reply:
previously added like that and was advised to do it here since it's a minor update

Reply:
fixed and the PR updated